kie-server-parent/kie-server-controller/kie-server-controller-client/pom.xml

```diff
@@ -43,7 +43,18 @@
       <artifactId>jakarta.websocket-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    
+    <dependency>
+      <groupId>jakarta.annotation</groupId>
+      <artifactId>jakarta.annotation-api</artifactId>
+    </dependency>
 
+    <dependency>
+      <groupId>jakarta.ws.rs</groupId>
+      <artifactId>jakarta.ws.rs-api</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    
```

Comment:
This is here twice, please remove it and look below ;)

Reply:
removed